### ISSUE TITLE
09_Operations_Pipelines.ipynb jumping

### ISSUE THREAD

**Issue by jbednar** [holoviz/holoviz#96]
If I zoom into the datashaded timeseries with outliers in notebook 09 and pan around a bit, eventally it gets stuck where it's alternating between two zoom levels even without my pointer moving at all:

![ezgif-3-a88e4f56d2](https://user-images.githubusercontent.com/1695496/45386944-9b725d80-b5e2-11e8-9be4-e8e7e9fcec30.gif)

Any theories why it's doing this?

**Comment by marcbernot** [comment]
I was able to reproduce this (though not right away).
To try to make sense out of this I narrowed the phenomenon down to this example
```
%%opts RGB {+framewise}
datashade(hv.Curve(np.random.randn(10000))).options(width=800,height=800)
```
Just by zooming once you get the cycling between two levels of zoom. If you don't get it, increase the width and height.
This is happening only with the +framewise option.
My theory (disclaimer :  I don't know a thing about the true machinery of datashader & holoviews) is that when you zoom you get through some range1 that triggers the rasterization1 for that range1, but the zoom finishes at range2 that triggers another rasterization2.
Then rasterization1 finishes and draws but since this changes the range2 to range1 this triggers the call rasterization1.bis.
Then rasterization2 finishes and draws and triggers rasterization2.bis, then rasterization1.bis finishes, draws, etc.

This scenario is also compatible with the fact that with width=height=1500 I was able to get cycling between three or four levels.
